cpp/cuproj/include/cuproj/operation/degrees_to_radians.cuh

```diff
@@ -57,7 +55,7 @@ class degrees_to_radians : operation<Coordinate> {
   __host__ __device__ Coordinate forward(Coordinate const& coord) const
   {
     using T = typename Coordinate::value_type;
-    return Coordinate{static_cast<T>(coord.x * DEG_TO_RAD), static_cast<T>(coord.y * DEG_TO_RAD)};
+    return Coordinate{coord.x * DEG_TO_RAD<T>, coord.y * DEG_TO_RAD<T>};
```

Comment:
I assume we expect other coordinate type than just `cuspatial::vec_2d`? Otherwise I thought we can just do `coord * DEG_TO_RAD<T>`.

Reply:
Yeah, I was concerned about that not working for POD data types, for example. But perhaps I shouldn't worry and just assume proper coordinate types.  We can't even use it directly with PJ_COORD because it's a union... https://proj.org/en/9.2/development/reference/datatypes.html#c.PJ_COORD

Reply:
We could probably provide an adaptor from PJ_COORD in the future.